.codecov.yml

```diff
@@ -11,11 +11,10 @@ coverage:
   status:
     project:
       default:
-        against: parent
-        target: auto
         threshold: 0.5%
-        branches:
-          - master
+    patch:
+      default:
+        threshold: 0.5%
```

Comment:
What about considering some of the parameters in the `coverage.status` section here:

https://docs.codecov.io/docs/codecovyml-reference

Reply:
That's what this PR does. Any others besides "threshold"?

Reply:
I've set `target` to 80%.